continuousprint/static/js/continuousprint.js

```diff
@@ -35,6 +35,7 @@ $(function() {
 				},
 				success:function(r){
 					if (r.queue.length > 0) {
+						$('#queue_list').html("");
```

Comment:
This is done earlier in the function.

Reply:
The first call to that statement clears the screen of the current jobs. The one I put in place on the success callback of the the Ajax call prevents and issue when two fetch ajax calls can return one after another can cause duplicate entries in the queue list. I am able to produce this issue when I tested with a 1 line Goode file.